packages/cosmwasm-typescript-gen/.eslintrc.js

```diff
@@ -0,0 +1,71 @@
+module.exports = {
+  plugins: ['prettier'],
+  extends: ['eslint:recommended', 'prettier'],
+  parser: '@babel/eslint-parser',
+  parserOptions: {
+    ecmaVersion: 11,
+    requireConfigFile: false,
+    sourceType: 'module',
+    ecmaFeatures: {
+      jsx: true
+    }
+  },
```

Comment:
GitHub is detecting some linting issues. Is it maybe some miss-alignment between this .eslintrc with the one configured in the pipeline or it can be fixed with fixlint ?